<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zchxMainWindow</class>
 <widget class="QMainWindow" name="zchxMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	color:red;
	font-weight:bold;
	font-size:18;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Ais_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string> AIS</string>
    </property>
   </widget>
   <widget class="QMenu" name="Radar_menu">
    <property name="title">
     <string> 雷达</string>
    </property>
   </widget>
   <widget class="QMenu" name="Log_menu">
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QMenu" name="Help_menu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单栏选项</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>:</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="Ais_menu"/>
   <addaction name="Radar_menu"/>
   <addaction name="Log_menu"/>
   <addaction name="Help_menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>开机</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
